documentation.go

```diff
@@ -434,7 +436,8 @@ func (d *documentationCommand) formatFlags(c Command, info *Info) string {
 			}
 			theFlags += fmt.Sprintf("`--%s`", f.Name)
 		}
-		formatted += fmt.Sprintf("| %s | %s | %s |\n", theFlags, fs[0].DefValue, fs[0].Usage)
+		formatted += fmt.Sprintf("| %s | %s | %s |\n", theFlags, fs[0].DefValue,
```

Comment:
We have to escape the default values too for some cases. In juju the `create-backup` command docks returns
```md
| `--filename` | juju-backup-<date>-<time>.tar.gz | Download to this file |
```